### ISSUE TITLE
Any though on when 0.0.6 final will be published?

### ISSUE THREAD

**Issue by superjulius** [clarkis117/Mime-Detective#28]
The latest version really looks nice. I know that this is a personal project but do you have any idea of when you will reach 0.0.6 final?

What remain to be done? And if you need some help, maybe add issues for the community to help you out.

Keep on the good work!

**Comment by clarkis117** [comment]
Right, apologies for my absence, but I plan to knock this out tomorrow. I've been occupied with moving and changing jobs over the past couple months.

**Comment by sandrock** [comment]
Hello @clarkis117,

I think 0.0.6 is doing quite well. The beta status on nuget prevent many users from getting the latest and better bytes (ahah). Can you consider making it stable?

**Comment by pluggles** [comment]
Any updates on a stable release of 0.0.6?

**Comment by 1100594** [comment]
Hi @clarkis117,

Any updates on this?

Best regards,

**Comment by sandrock** [comment]
Hey there @clarkis117 
We would really appreciate a stable release of the project.
Is there anything that prevents you from doing it? Can we help?
